### PR TITLE
Add enum

### PR DESCRIPTION
**Pull request by JuliaZharova**

### CODE REVIEW

File: Lecture-9/src/com/company/Main.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Main {
+
+    public static void main(String[] args) {
+	    Dog dog1 = new Dog("Dog1", Breed.Bulldog,5);
+	    Dog dog2 = new Dog("Dog2", Breed.Beagle,5);
+        Dog dog3 = new Dog("Dog3", Breed.Spaniel,5);
```
Comment (ltana) [inline Lecture-9/src/com/company/Main.java:11]: please use different age

File: Lecture-9/src/com/company/Main.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            || dog1.name.equals(dog3.name)
+            || dog2.name.equals(dog3.name)) {
+            System.out.println("There are at least 2 dogs with the same name");
+        } else {
+            System.out.println("There are no dogs with the same name");
+        }
+
+        ArrayList<Dog> oldestDogs = new ArrayList<>();
+        oldestDogs.add(dog1);
+        if(oldestDogs.get(0).age < dog2.age) {
```
Comment (ltana) [inline Lecture-9/src/com/company/Main.java]: is it possible here to combine if and else if blocks?

**Review by ltana — CHANGES_REQUESTED** [review]

File: Lecture-9/src/com/company/Main.java (hunk 1 above)
Comment (JuliaZharova) [reply Lecture-9/src/com/company/Main.java:11]: fixed

File: Lecture-9/src/com/company/Main.java (hunk 2 above)
Comment (JuliaZharova) [reply Lecture-9/src/com/company/Main.java]: fixed

**Review by ltana — APPROVED** [review]
can be merged

**Pull request merged**